### PR TITLE
Correct usage of Timer and Ticker

### PR DESCRIPTION
**Pull request by czs007**
related : #22222

### CODE REVIEW

File: internal/datanode/binlog_io.go [hunk 1]
```diff
@@ -73,17 +74,20 @@ func (b *binlogIO) download(ctx context.Context, paths []string) ([]*Blob, error
 
 	g, gCtx := errgroup.WithContext(ctx)
 	g.Go(func() error {
+		errStart = retry.Do(gCtx, func() error {
```
Comment (bigsheeper) [inline internal/datanode/binlog_io.go:77]: remove this

File: internal/datacoord/channel_manager.go [hunk 2]
```diff
@@ -249,6 +249,7 @@ func (c *ChannelManager) unwatchDroppedChannels() {
 // NOT USED.
 func (c *ChannelManager) bgCheckChannelsWork(ctx context.Context) {
 	timer := time.NewTicker(bgCheckInterval)
+	defer timer.Stop()
```
Comment (xiaofan-luan) [inline internal/datacoord/channel_manager.go:252]: should we change it to ticker?

**Comment by xiaofan-luan** [comment]
others all looks good to me

File: internal/datacoord/channel_manager.go (hunk 2 above)
Comment (czs007) [reply internal/datacoord/channel_manager.go:252]: yes, rename it to ticker

**Comment by XuanYang-cn** [comment]
LGTM

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**